### DISCUSSION TITLE
A fabric-based graphics editor applies mitu-editor👋

### DISCUSSION THREAD

**Discussion by MrXujiang** [fabricjs/fabric.js#7461] [Show and tell]
Thank you very much for fabric, I developed a graphics editor mitu based on it, hoping it will help fabric make the community stronger.
<img alt="License: MIT" src="http://cdn.dooring.cn/dr/mitu.png" />

[mitu-fabric](https://github.com/H5-Dooring/mitu-editor)

### Features
- [x] The underlying graphics component
- [x] Component controller, support for deletion, control handle, rotation, element multi-select
- [x] Save the picture
- [x] Save the picture template
- [x] Graphic property editor
- [x] Canvas control
- [x] Template list, one-click import footage template
- [x] Empty the canvas
- [x] Live preview
- [x] Color reader

### Todo
- [x] Undo redoing
- [x] Canvas background settings
- [x] Enrich the graphics component library
- [x] Modular interface
- [x] Resolve PSD

**Comment by asturur** [comment]
Nice, is it open source?

**Reply by MrXujiang** [reply]
yes

**Comment by jandesign-lee** [comment]
你好能够打开psd文件进行编辑，并储存为PSD文件下载到本地吗

**Comment by nihaojob** [comment]
Can look at this project ，  vue-fabric-editor。

https://github.com/nihaojob/vue-fabric-editor

![image](https://user-images.githubusercontent.com/13534626/221137519-ac00e126-9943-4f84-8368-048d7678104b.png)
